jobclient/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -883,6 +903,12 @@ public static JSONObject jsonizeJob(Collection<Job> jobs)
                     jobBuilder.addUri(FetchableURI.parseFromJSON(urisJson.getJSONObject(j)));
                 }
             }
+            if (json.has("constraints")) {
+                JSONArray constraintsJson = json.getJSONArray("constraints");
+                for (int j = 0; j < constraintsJson.length(); j++) {
```

Comment:
We can parse out the operators and call the `operator.parseFrom()` here and avoid the need for the `Constraints` class.